src/test/scala/au/com/agiledigital/toolform/reader/ProjectReaderTest.scala

```diff
@@ -0,0 +1,47 @@
+package au.com.agiledigital.toolform.reader
+
+import java.io.File
+
+import au.com.agiledigital.toolform.model.Project
+import org.scalatest.Inside.inside
+import org.scalatest._
+
+class ProjectReaderTest extends FlatSpec with Matchers {
+
+  val testFile = pathToFile("/testprojects/realworldsample/environment.conf")
+  val emptyFile = pathToFile("/errors/empty.conf")
+  val malformedFile = pathToFile("/errors/malformed.conf")
+
+  def pathToFile(pathToFile: String): File = {
+    val url = getClass.getResource(pathToFile)
+    val file = new File(url.toURI)
+    file
+  }
+
+  "reading a valid project" should "return a Project model" in {
+    val result = ProjectReader.readProject(testFile)
+    inside(result) {
+      case Right(project) => project shouldBe a[Project]
+      case Left(error)    => fail(error.message)
+    }
+  }
+
+  "inspect blank file" should "display error string" in {
+    val result = ProjectReader.readProject(emptyFile)
+    result.left.get.message should startWith("Failed to read project")
```

Comment:
`result.left.get` will explode unhelpfully if it isn't a `Left`. See http://www.scalatest.org/user_guide/other_goodies#eitherValues

Reply:
Ah, this is good.  `value` it is!